lib/elements/dom-bind.html

```diff
@@ -61,16 +61,19 @@
         this.__children = null;
       }
 
-      // assumes only one observed attribute
+      /** @returns {void} */
       attributeChangedCallback() {
+        // assumes only one observed attribute
         this.mutableData = true;
       }
 
+      /** @returns {void} */
       connectedCallback() {
         this.style.display = 'none';
         this.render();
       }
 
+      /** @returns {void} */
```

Comment:
`@return` is only necessary if the function takes arguments

Reply:
I added it because otherwise in TypeScript we have to output returns `any` in this case, because we have no idea what the return type is.